docs/reference/configuration-examples.md

```diff
@@ -96,9 +100,9 @@ recyclarr sonarr --config sonarr-release-profiles.yml sonarr-quality-definition.
 
 ## Update Sonarr v3 and v4 instances in a single YAML config
 
-:::danger
+:::caution
 
-Custom Formats are only supported in Sonarr v4 which <u>**is prerelease software**</u>!
+Custom Formats are only supported in Sonarr v4 which <u>is **beta** software</u>!
 
 If you are new to the TRaSH Guides, I highly recommend you start with Sonarr v3 Release Profiles
 instead.
```

Comment:
Let's go ahead and remove the whole admonition. That's what I was planning, anyway. I don't think your change is bad, but I also don't think I care too much about something being beta. Not to the extent that I want to have to worry about coming back here *again* to remove it later.

Reply:
Actually, I realized I already had a local commit that I forgot to push where I removed this admonition + one other I don't see altered here. I'd say go ahead and rebase/merge with `master` and you should be able to discard this hunk.